src/main/java/org/folio/des/service/FolioTenantService.java

```diff
@@ -55,6 +60,7 @@ protected void afterTenantUpdate(TenantAttributes tenantAttributes) {
       oldJobDeleteScheduler.scheduleOldJobDeletion(context.getTenantId());
       kafka.createKafkaTopics();
       kafka.restartEventListeners();
+      LegacyBursarMigrationUtil.recreateLegacyJobs(bursarExportLegacyJobService, jobService);
```

Comment:
all other places in this method uses injected service, so code decoupled, testable, can be mocked etc.
Better to use service here as well instead of utility class.

Reply:
Should I then refactor `LegacyBursarMigrationUtil` to something like `BursarMigrationService` class?

Reply:
yes, it would be better, in this case our components will be decoupled